src/styles/themes/avatar.ts

```diff
@@ -0,0 +1,29 @@
+import type { ThemeOption } from '@types'
+
+type AvatarKeys = keyof typeof avatar
+type AvatarValues = typeof avatar[AvatarKeys]
+
+export type Avatar = ThemeOption<AvatarKeys, AvatarValues>
+
+export const avatar = {
+  large: `width: 112px; height: 112px; 
+  & > img {
+    width: 55px;
+    height: 55px;
+  }`,
+  medium: `width: 80px; height: 80px;
+  & > img {
+    width: 40px;
+    height: 40px;
+  }`,
+  small: `width: 46px; height: 46px;
+  & > img {
+    width: 23px;
+    height: 23px;
+  }`,
+  xsmall: `width: 32px; height: 32px;
+  & > img {
+    width: 16px;
+    height: 16px;
+  }`
```

Comment:
```suggestion
export const avatar = {
  large: `
    width: 112px;
    height: 112px; 
    &  svg {
      width: 55px;
      height: 55px;
    }`,
  medium: `width: 80px; height: 80px;
  & svg {
    width: 40px;
    height: 40px;
  }`,
  small: `width: 46px; height: 46px;
  & svg {
    width: 23px;
    height: 23px;
  }`,
  xsmall: `width: 32px; height: 32px;
  & svg {
    width: 16px;
    height: 16px;
  }`
```

`& > img`에서 `& svg`로 변경되어야 적용이 됨니다!
+ 줄 바꿈 large처럼 변경 부탁드림니둥 ..ㅎㅎㅎ